actions/vmwarelib/actions.py

```diff
@@ -79,6 +84,39 @@ def _connect(self, vsphere):
         atexit.register(connect.Disconnect, si)
         return si
 
+    def _connect_rest(self, vsphere):
+        connection = self._get_connection_info(vsphere)
+
+        try:
+            session = requests.Session()
+            session.verify = False
+            session.auth = (connection['user'], connection['passwd'])
+
+            login_url = "https://%s/rest/com/vmware/cis/session" % connection['host']
+
+            session.post(login_url)
+
+        except Exception as e:
+            raise Exception(e)
+
+        return session
+
+    def _rest_api_call(self, vsphere, api_endpoint, api_verb, post_params=None):
```

Comment:
post_params is not a good name for this variable, instead it should be `payload`